jobscheduler/scheduler-component/src/webapp/WEB-INF/components.xml

```diff
@@ -213,6 +213,12 @@
                    <property name="descriptionResourceKey" value="logger.description"/>
                    <property name="defaultValue" value="org.sakaiproject.component.app.scheduler.jobs.logmessage.LogMessageJob"/>
                </bean>
+               <bean class="org.sakaiproject.component.app.scheduler.jobs.SpringConfigurableJobProperty">
+                   <property name="required" value="true"/>
```

Comment:
This property doesn't "seem" like it's required?

Reply:
Nice catch Matt, fixed.